### PR TITLE
ER-929 - Viewing pending, live and closed vacancies

### PR DESCRIPTION
**Pull request by chrisjensenuk**
- I've cut-and-pasted a lot of Manage vacancy code however at the moment some of it isn't being used. This will be activated in future tickets.  (I thought it was simpler to do this than to only implement whats needed for this ticket and then try and patch the missing manage functionality back in again at a later date)
- To satisfy the ticket I've temporarily added a link in the `Dashboard.cshtml` to only allow the viewing of vacancies.  When we start to do the manage vacancy tickets we'll need to delete this link and uncomment the link above it to manage the vacancy.
- Includes the changes that I've added to the release hotfix 616fff7260bac05f4f17992eef938be8bd1fd771

### CODE REVIEW

File: src/Employer/Employer.Web/Orchestrators/VacancyManageOrchestrator.cs [hunk 1]
```diff
@@ -52,7 +52,7 @@ public async Task<ManageVacancyViewModel> GetManageVacancyViewModel(Vacancy vaca
             viewModel.CanShowEditVacancyLink = vacancy.CanExtendStartAndClosingDates;
             viewModel.CanShowCloseVacancyLink = vacancy.CanClose;
 
-            var vacancyApplicationsTask =  await _client.GetVacancyApplicationsAsync(vacancy.VacancyReference.Value.ToString());
+            var vacancyApplicationsTask =  await _vacancyClient.GetVacancyApplicationsAsync(vacancy.VacancyReference.Value.ToString());
```
Comment (shomavg) [inline src/Employer/Employer.Web/Orchestrators/VacancyManageOrchestrator.cs]: can the names of the variables me more specific , something like, _employerclient and _recruiterclient?

Comment (chrisjensenuk) [reply src/Employer/Employer.Web/Orchestrators/VacancyManageOrchestrator.cs]: I agree however this is the pattern that's been used across the codebase so I'm just fitting in.

Comment (shomavg) [reply src/Employer/Employer.Web/Orchestrators/VacancyManageOrchestrator.cs]: Do we need IEmployerVacancyClient _client ? As its not being used ?

Comment (chrisjensenuk) [reply src/Employer/Employer.Web/Orchestrators/VacancyManageOrchestrator.cs]: 👍 Will remove

File: src/Employer/Employer.Web/Orchestrators/VacancyViewOrchestrator.cs [hunk 2]
```diff
@@ -138,7 +138,7 @@ private ViewVacancy GetDisplayViewModelForClosedVacancy(Vacancy vacancy)
         private void PopulateViewModelWithApplications(Vacancy vacancy, DisplayVacancyApplicationViewModel viewModel)
         {
             var mappedDisplayVacancyViewModelTask = _vacancyDisplayMapper.MapFromVacancyAsync(viewModel, vacancy);
-            var vacancyApplicationsTask = _client.GetVacancyApplicationsAsync(vacancy.VacancyReference.Value.ToString());
+            var vacancyApplicationsTask = _vacancyClient.GetVacancyApplicationsAsync(vacancy.VacancyReference.Value.ToString());
```
Comment (shomavg) [inline src/Employer/Employer.Web/Orchestrators/VacancyViewOrchestrator.cs]: Again. Can we remove the IEmployerVacancyClient _client from the constructor?

Comment (chrisjensenuk) [reply src/Employer/Employer.Web/Orchestrators/VacancyViewOrchestrator.cs]: 👍

**Pull request merged**